.github/workflows/test.yml

```diff
@@ -20,6 +20,17 @@ jobs:
         run: python -m pip install --upgrade pip
       - name: Install dependencies
         run: |
-          pip install .[dev]
+          if [ "${{ matrix.python-version }}" = "3.7" ]; then
+            pip install "tutor==15.0.0"
```

Comment:
Oh wow you want to be backward-compatible with v15? That's ambitious.

Reply:
:)) Is that bad? We can cover more tutor users by leaving behind a couple of filters like `APP_PUBLIC_HOSTS`.